### PR TITLE
Generate cc and hdr files for person_detection_benchmark.

### PR DESCRIPTION
**Pull request by njeffrie**
This is required since the old person_detect_benchmark depended on a deleted file `tensorflow/lite/micro/examples/person_detect_model_data.h` which is now generated as part of the build.

BUG=b/196065539
FIXES=b/196879878

### CODE REVIEW

File: tensorflow/lite/micro/tools/make/Makefile [hunk 1]
```diff
@@ -465,7 +465,6 @@ include $(MAKEFILE_DIR)/third_party_downloads.inc
 THIRD_PARTY_DOWNLOADS :=
 $(eval $(call add_third_party_download,$(GEMMLOWP_URL),$(GEMMLOWP_MD5),gemmlowp,))
 $(eval $(call add_third_party_download,$(RUY_URL),$(RUY_MD5),ruy,))
-$(eval $(call add_third_party_download,$(PERSON_MODEL_URL),$(PERSON_MODEL_MD5),person_model_grayscale,))
 RESULT := $(shell $(MAKEFILE_DIR)/person_detection_int8_download.sh ${MAKEFILE_DIR}/downloads $(CO_PROCESSOR))
```
Comment (advaitjain) [inline tensorflow/lite/micro/tools/make/Makefile:468]: remove?

File: tensorflow/lite/micro/tools/make/third_party_downloads.inc [hunk 2]
```diff
@@ -39,9 +39,6 @@ KISSFFT_MD5="438ba1fef5783cc5f5f201395cc477ca"
 RUY_URL="https://github.com/google/ruy/archive/d37128311b445e758136b8602d1bbd2a755e115d.zip"
 RUY_MD5="abf7a91eb90d195f016ebe0be885bb6e"
 
-PERSON_MODEL_URL := "https://storage.googleapis.com/download.tensorflow.org/data/tf_lite_micro_person_data_grayscale_2020_05_27.zip"
-PERSON_MODEL_MD5 := "55b85f76e2995153e660391d4a209ef1"
-
 PERSON_MODEL_INT8_URL := "https://storage.googleapis.com/download.tensorflow.org/data/tf_lite_micro_person_data_int8_grayscale_2020_12_1.zip"
 PERSON_MODEL_INT8_MD5 := "e765cc76889db8640cfe876a37e4ec00"
```
Comment (advaitjain) [inline tensorflow/lite/micro/tools/make/third_party_downloads.inc:43]: remove this as well?

**Review by advaitjain — COMMENTED** [review]
I'm working on a change that does this and fixes the ethos_u + vela build. That will take a bit longer so let's get this change merged first.

File: tensorflow/lite/micro/tools/make/Makefile (hunk 1 above)
Comment (njeffrie) [reply tensorflow/lite/micro/tools/make/Makefile:468]: Done

**Review by advaitjain — APPROVED** [review]

File: tensorflow/lite/micro/tools/make/third_party_downloads.inc (hunk 2 above)
Comment (njeffrie) [reply tensorflow/lite/micro/tools/make/third_party_downloads.inc:43]: Remove once vela is updated

**Pull request merged**